### PR TITLE
Feature/25 evaluation crud

### PR DESCRIPTION
**Pull request by JuanluRM**
Added Student Evaluation and Evaluation Type CRUD options plus the validator for the grading system. I haven't identified any other necessary validators aside of the mentioned before.

### CODE REVIEW

**Review by pedlopruz — DISMISSED** [review]

File: nexong/api/routers.py [hunk 1]
```diff
@@ -2,12 +2,23 @@
...
+    prefix="student-evaluation",
+    viewset=StudentEvaluationApiViewSet,
+    basename="studentevaluation",
+)
+router_api.register(
+    prefix="evaluation-type",
+    viewset=EvaluationTypeApiViewSet,
+    basename="evaluationtype",
+)
+router_api.register(prefix="volunteer", viewset=EventApiViewSet, basename="volunteer")
```
Comment (Ahydul) [inline nexong/api/routers.py:24]: There is a mistake here but it has nothing to do with your task. I would delete this because it is related to another PR

**Review by pabpercab1 — APPROVED** [review]
Looks good. Great work with the Evaluation!

**Review by FelixoGudiel — APPROVED** [review]
Works as expected. Good job!

File: nexong/api/routers.py (hunk 1 above)
Comment (JuanluRM) [reply nexong/api/routers.py]: Line was erased as requested

**Pull request merged**